pylint/checkers/refactoring.py

```diff
@@ -675,16 +682,38 @@ def _check_consider_using_comprehension_constructor(self, node):
                 message_name = "consider-using-set-comprehension"
                 self.add_message(message_name, node=node)
 
+    def _check_consider_using_generator(self, node):
+        checked_call = ["any", "all"]
```

Comment:
Why only `any` and `all`? The same basic premise applies to `list`, `set`, and `tuple` too:
```
>>> tuple(list(range(10)))
(0, 1, 2, 3, 4, 5, 6, 7, 8, 9)
>>> tuple(range(10))
(0, 1, 2, 3, 4, 5, 6, 7, 8, 9)
```

Reply:
See answer here: https://github.com/PyCQA/pylint/pull/3309#discussion_r576683109